### PR TITLE
lesson 02 task

### PR DESCRIPTION
**Pull request by MariiaChm**

### CODE REVIEW

File: lesson_02/8081/main.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+from flask import Flask, request
+import os
+import sys
+import requests
+import json
+
```
Comment (mikolaz27) [inline lesson_02/8081/main.py]: remove it and add init.py to the folders

File: lesson_02/8081/main.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# setting path
+sys.path.append('../')
+
+from common_methods import create_or_clear_directory
+
+app = Flask(__name__)
+
+AUTH_TOKEN = os.environ['AUTH_TOKEN']
```
Comment (mikolaz27) [inline lesson_02/8081/main.py]: better to move it to settings module

File: lesson_02/8081/main.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            i += 1
+            response = requests.get(
+                url='https://fake-api-vycpfa6oca-uc.a.run.app/sales',
+                params={'date': dt, 'page': i},
+                headers={'Authorization': AUTH_TOKEN},
+            )
+            print("Response status code:", response.status_code)
+
+            # find last page
+            if response.status_code == 404:
```
Comment (mikolaz27) [inline lesson_02/8081/main.py]: move this condition to the while

File: lesson_02/8081/main.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+
+@app.route('/', methods=['POST'])
+def handle_post_request() -> str:
+    # get data from post request
+    data: dict = request.json
+
+    raw_dir: str = data.get('raw_dir')
+    assert raw_dir, 'raw_dir is not set'
+    dt: str = data.get('date')
```
Comment (mikolaz27) [inline lesson_02/8081/main.py:14]: raise flask error as a response

File: lesson_02/8082/main.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+sys.path.append('../')
+from common_methods import create_or_clear_directory
+
+
+app = Flask(__name__)
+
+AUTH_TOKEN = os.environ['AUTH_TOKEN']
+
+SCHEMA = {
```
Comment (mikolaz27) [inline lesson_02/8082/main.py]: better to have separate file for schema

File: lesson_02/common_methods.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import os
```
Comment (mikolaz27) [inline lesson_02/common_methods.py:1]: just common

File: lesson_02/8082/main.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+
+@app.route('/', methods=['POST'])
+def handle_post_request() -> str:
+    # get data from post request
+    data: dict = request.json
+
+    raw_dir: str = data.get('raw_dir')
+    assert raw_dir, 'raw_dir is not set'
```
Comment (mikolaz27) [inline lesson_02/8082/main.py]: the same as before

File: lesson_02/8081/main.py (hunk 1 above)
Comment (MariiaChm) [reply lesson_02/8081/main.py]: I am not sure if I got it correctly, please check

File: lesson_02/8081/main.py (hunk 2 above)
Comment (MariiaChm) [reply lesson_02/8081/main.py]: I am not sure if I got it correctly, please check

**Pull request merged**